WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -665,7 +665,7 @@ public void onStart() {
 
     @Override
     public void setTitle(final String title) {
-        mToolbarTitle = title;
+        mToolbarTitle = (title.equals("")) ? "WordPress" : title;
```

Comment:
We can use `TextUtils.isEmpty` instead of directly checking for empty string.

We also need to localize the string. I suggest adding a new string resource, probably something like `wordpress_as_site_title` to make it clear how it's used to give information to the translators.